packages/nc-gui-v2/components/smartsheet-column/LinkedToAnotherRecordOptions.vue

```diff
@@ -0,0 +1,109 @@
+<script setup lang="ts">
+import type { TableType } from 'nocodb-sdk'
+import { ModelTypes, MssqlUi, SqliteUi } from 'nocodb-sdk'
+import { useColumnCreateStoreOrThrow } from '#imports'
+import { MetaInj } from '~/context'
+import MdiPlusIcon from '~icons/mdi/plus-circle-outline'
+import MdiMinusIcon from '~icons/mdi/minus-circle-outline'
+
+const { formState, validateInfos, onDataTypeChange, setAdditionalValidations } = $(useColumnCreateStoreOrThrow())
+const { tables, sqlUi } = $(useProject())
+const meta: TableType = $(inject(MetaInj))
+
+console.log(tables)
+
+setAdditionalValidations({
+  childId: [{ required: true, message: 'Required' }],
+})
+
+const onUpdateDeleteOptions = sqlUi === MssqlUi ? ['NO ACTION'] : ['NO ACTION', 'CASCADE', 'RESTRICT', 'SET NULL', 'SET DEFAULT']
+
+if (!formState.parentId) formState.parentId = meta.id
```

Comment:
```js
formState = {
  parentId: meta.id,
  // same for L22 - L32
  ...formState,
}
```

Reply:
@wingkwong This will decrease the readability of this code section. What is benefit of doing this way?